insights-ansible-playbook-verifier.spec

```diff
@@ -0,0 +1,42 @@
+Name:     insights-ansible-playbook-verifier
+Version:  0.1.0
+Release:  0%{dist}
+Summary:  Ansible playbook verifier for Red Hat Insights
+License:  MIT
+
+URL:      https://github.com/RedHatInsights/%{name}
+# Source0:  %{url}/archive/refs/tags/v%{version}.tar.gz
+Source0:  %{name}-%{version}.tar.gz
+
+BuildArch: noarch
+Requires:  python3
+Requires:  python3-pyyaml
```

Comment:
those two should added automatically; at least were for me when building locally on f39:
```
/usr/bin/python3
python(abi) = 3.12
python3.12dist(pyyaml)
[...]
```
(from the output of `rpm -qfR <built>.rpm`)